### ISSUE TITLE
Up Next marking an unwatched next episode as watched

### ISSUE THREAD

**Issue by nikhillfc** [im85288/service.upnext#246]
Hi, I use Embycon as my main video source, but the native Play Next on it isn't as great as Up Next. So I use Up Next and hit Watch Next Episode just before the end of the current episode, and the next episode plays just fine... however if I immediately stop the video after that, that episode has been marked as watched. The 'Next Episodes' section of Embycon now shows the subsequent episode in the Next Up episodes list.

So I thought both Embycon and Up Next are sending next episodes command one after the other, but don't think that's the case as I've disabled Play Next inside Embycon. 

tl;dr: I think whenever I click on Watch Next Episode through the Up Next prompt, it is being marked as 'Watched' as soon as it starts.

Log File: https://paste.kodi.tv/ilowumuyiz

**Comment by im85288** [comment]
Hi,

It sounds like this is one to report to the EmbyCon developer. Marking as watched on Emby only happens via the EmbyCon addon.

**Comment by nikhillfc** [comment]
Oh... thank you for looking into it. I will post it there!
